projects/aca-shared/src/lib/components/page-layout/page-layout-content.component.ts

```diff
@@ -24,6 +24,7 @@
 
 import { Component, ViewEncapsulation, ChangeDetectionStrategy, Input, HostBinding } from '@angular/core';
 
+// @deprecated Use `.aca-page-layout-content` CSS selector instead
```

Comment:
@DenysVuika can you raise a jira to don't forget remove these deprecated components in the future?

Reply:
created https://alfresco.atlassian.net/browse/ACS-5636